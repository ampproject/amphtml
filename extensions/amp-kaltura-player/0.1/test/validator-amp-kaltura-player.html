<!--
  Test Description:
  Tests for the amp-youtube tag. See the inline comments.
-->
<!doctype html>
<html âš¡ lang="en">
<head>
  <meta charset="utf-8">
  <link rel="canonical" href="./regular-html-version.html">
  <meta name="viewport" content="width=device-width,minimum-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
  <script async custom-element="amp-kaltura-player" src="https://cdn.ampproject.org/v0/amp-kaltura-player-latest.js"></script>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
</head>
<body>
  <!-- valid example from the documentation -->
  <amp-kaltura-player
     data-uiconf="33502051"
     data-partner="1281471"
     data-entryid="1_3ts1ms9c"
     data-param-streamerType = "auto"
     layout="responsive" width="480" height="270">
  </amp-kaltura-player>

  <!-- valid kaltura ce example -->
  <amp-kaltura-player
    data-service-url="front.video.funke.press"
    data-uiconf="23464665"
    data-partner="106"
    data-entryid="0_b87xdluw"
    data-param-streamerType="auto"
    layout="responsive" width="480" height="270">
  </amp-kaltura-player>

  <!-- invalid example: data-partner is missing -->
  <amp-kaltura-player width="480" height="270"></amp-kaltura-player>
</body>
</html>
