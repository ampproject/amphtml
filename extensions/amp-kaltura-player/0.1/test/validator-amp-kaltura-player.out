FAIL
|  <!--
|    Copyright 2015 The AMP HTML Authors. All Rights Reserved.
|
|    Licensed under the Apache License, Version 2.0 (the "License");
|    you may not use this file except in compliance with the License.
|    You may obtain a copy of the License at
|
|        http://www.apache.org/licenses/LICENSE-2.0
|
|    Unless required by applicable law or agreed to in writing, software
|    distributed under the License is distributed on an "AS-IS" BASIS,
|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
|    See the License for the specific language governing permissions and
|    limitations under the license.
|  -->
|  <!--
|    Test Description:
|    Tests for the amp-youtube tag. See the inline comments.
|  -->
|  <!doctype html>
|  <html âš¡>
|  <head>
|    <meta charset="utf-8">
|    <link rel="canonical" href="./regular-html-version.html">
|    <meta name="viewport" content="width=device-width,minimum-scale=1">
|    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
|    <script async custom-element="amp-kaltura-player" src="https://cdn.ampproject.org/v0/amp-kaltura-player-latest.js"></script>
|    <script async src="https://cdn.ampproject.org/v0.js"></script>
|  </head>
|  <body>
|    <!-- valid example from the documentation -->
|    <amp-kaltura-player
|       data-uiconf="33502051"
|       data-partner="1281471"
|       data-entryid="1_3ts1ms9c"
|       data-param-streamerType = "auto"
|       layout="responsive" width="480" height="270">
|    </amp-kaltura-player>
|
|    <!-- valid kaltura ce example -->
|    <amp-kaltura-player
|      data-service-url="front.video.funke.press"
|      data-uiconf="23464665"
|      data-partner="106"
|      data-entryid="0_b87xdluw"
|      data-param-streamerType="auto"
|      layout="responsive" width="480" height="270">
|    </amp-kaltura-player>
|
|    <!-- invalid example: data-partner is missing -->
|    <amp-kaltura-player width="480" height="270"></amp-kaltura-player>
>>   ^~~~~~~~~
amp-kaltura-player/0.1/test/validator-amp-kaltura-player.html:51:2 The mandatory attribute 'data-partner' is missing in tag 'amp-kaltura-player'. (see https://amp.dev/documentation/components/amp-kaltura-player)
|  </body>
|  </html>
